roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,12 @@
 dependencies:
   - pip
+  - role: logrotate
+    logrotate_configs:
```

Comment:
The idea is to reuse the same role on different hosts, each would have a unique list of files to rotate.